src/components/Home.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+
```

Comment:
These head and body tags are also in index.html. 

The way that the frontend loads is like this
- index.html loads and defines the base container (app) and includes main.ts
- main.ts bootstraps the frontend framework Vue.js
- Vue.js frontend router determines which view / route to load
- The view is loaded, & any layouts are applied.
- Components used by the view are loaded.

So in essence, what this is doing is like

```html
<html>
<head>
  <!-- Some bullshit -->
</head>
<body>
  <div id="app">
    <div id="app">
      <head>
      </head>
      <body>
        <!-- Your sidebar and etc -->
      </body>
    </div>
  </div>
</body>

</html>
```

Components should be more focused on individual components.

i.e. you'd have a sidebar, a content thing, etc. Quasar provides most of these for your convenience, but you'll certainly end up creating custom or derived versions of those components.